run-google-java-format.py

```
@@ -118,8 +116,8 @@ def under_git(directory: Path, filename: str) -> bool:
 
 def urlretrieve(url: str, filename: Path) -> None:
     """Like urllib.urlretrieve."""
-    with urlopen(url) as in_stream, pathlib.Path(filename).open("wb") as out_file:
-        copyfileobj(in_stream, out_file)
+    with urlopen(url) as in_stream, filename.open("wb") as out_file:
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)